ImperatorToCK3/Source/Imperator/Countries/Country.h

```diff
@@ -48,6 +49,7 @@ namespace ImperatorWorld
 			void registerKeys();
 
 			int countryID = 0;
+			std::optional<int> monarch;
```

Comment:
If I told you different before I apologize. Please move these back to int. It's the same as 
int monarch = 0;

Reply:
0 is a valid character ID in Imp, so it won't work.

Reply:
but std::optional<int> is identical to int=0. Move it ti int=-1 then.

Reply:
Why is optional a bad option here? I'd rather use `std::optional<unsigned int>` to make clearer that IDs are unsigned in the save.

Reply:
✔️